src/graph.py

```diff
@@ -752,6 +741,105 @@ def perform_GO_enrichment_on_clusters(nxgraph, clustering):
     """
     pass
 
+
+###############################################################################
+            #######            Results           #######
+###############################################################################
+
+def summarize_grid_search(results, mode, top_n=False): 
+    """
+    Summarizes results of `_grid_randomization` into a matrix where each row is a gene 
+    and each column is a parameter run. If summarizing "membership", entries will be 0 or 1
+    indicating whether or not a node appeared in each experiment. If summarizing "robustness"
+    or "specificity", entries indicate robustness or specificity values for each experiment. 
+    Also, node attributes columns are added for plotting purposes.
+
+    Arguments:
+        results (list of tuples): Results of `_grid_randomization` (paramstring, forest, augmented forest)
```

Comment:
> results (list of tuples): Results of `_grid_randomization` (paramstring, forest, augmented forest)

`_grid_randomization` now has a different name, no?

maybe also change 

> `(paramstring, forest, augmented forest)`

to `{'paramstring': { 'forest': object, 'augmented forest': object}}

since that's the format it actually needs to be in. Speaking of, why does it need to be in that format again?

Reply:
eec5595640c22e74d17bbe6a09a3a715ebbb3567

For 2 reasons I can think of: 
1. `robust` networks will be added for each parameter, and dictionaries make this easy
2. When user wishes to output a specific network result, it's easy to find networks associated with a parameter set when using dics, but less convenient if using tuples (unless there's a clever way I'm unaware of)